WarOOP/Models/Army.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Threading.Channels;
+using WarOOP.Abstractions;
+
+namespace WarOOP.Models;
+
+public class Army : IArmy
+{
+    public List<Warrior> Units;
```

Comment:
Why is this property public? Any other class can reassign and change the collection